.travis.yml

```diff
@@ -1,6 +1,15 @@
 sudo: false
-language: haskell
+addons:
+  apt:
+    sources:
+        - hvr-ghc
+    packages:
+        - libgmp-dev
+        - ghc-8.0.1
```

Comment:
AWS builds with all compilers just fine. But aeson doesn't. I tried to downgrade it to 0.11, but it is still failing: https://travis-ci.org/rblaze/aws/builds/196095432